### PR TITLE
contentControllerReviwerCommunicant

### PR DESCRIPTION
**Pull request by JQUEVENNE**
probleme sur le content controller public function reviwer et publish

### CODE REVIEW

File: config/packages/security.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 security:
     encoders:
-        App\Entity\User: auto
+    App\Entity\User: auto
+    role_hierarchy:
+        ROLE_REVIWER: ROLE_USER
+        ROLE_COMMUNICANT: ROLE_REVIWER
+        ROLE_ADMIN: ROLE_COMMUNICANT
```
Comment (ttdepaille) [inline config/packages/security.yaml:7]: To Delete

File: config/packages/security.yaml [hunk 2]
```diff
@@ -29,4 +33,7 @@ security:
         ROLE_ADMIN:        ROLE_PUBLISHER
     access_control:
         - { path: ^/home, roles: ROLE_USER }
+        - { path: ^/rewiver, roles: ROLE_REWIVER }
```
Comment (ttdepaille) [inline config/packages/security.yaml:36]: Typo /reviewer

File: config/packages/security.yaml [hunk 3]
```diff
@@ -29,4 +33,7 @@ security:
...
+        - { path: ^/communicant, roles: ROLE_COMMUNICANT }
```
Comment (ttdepaille) [inline config/packages/security.yaml:37]: /publisher instead

File: src/Controller/ContentController.php [hunk 4]
```diff
@@ -87,12 +87,36 @@ public function edit(Request $request, Content $content): Response
      */
     public function delete(Request $request, Content $content): Response
     {
-        if ($this->isCsrfTokenValid('delete'.$content->getId(), $request->request->get('_token'))) {
+        if ($this->isCsrfTokenValid('delete' . $content->getId(), $request->request->get('_token'))) {
```
Comment (ttdepaille) [inline src/Controller/ContentController.php:90]: whitespaces?

**Comment by JQUEVENNE** [comment]
too much inconsistency

**Pull request closed without merging**